package.json

```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "@aws-sdk/client-dynamodb": "3.609.0",
     "@govuk-one-login/di-ipv-cri-common-express": "10.0.1",
-    "@govuk-one-login/frontend-analytics": "3.0.1",
+    "@govuk-one-login/frontend-analytics": "3.0.0",
```

Comment:
curious this has gone down a version?

Reply:
It's because common express is using 3.0.0 and the versions conflict. Think we might need to look at the dependabot PRs in CE!